### ISSUE TITLE
Different outputs when querying metric from Cyanite

### ISSUE THREAD

**Issue by avneradania** [pyr/cyanite#277]
I installed latest Cyanite and cannot see any graph in Grafana.
```
# curl 'http://127.0.0.1:18080/metrics?path=AIO.DIAG.diag2.cpu.cpu.usage&from=1495981080&to=1496067480'
```

Old Cyanite output:
```
{"from":1496056860,"to":1496067480,"step":60,"series":{"AIO.DIAG.diag2.cpu.cpu.usage":[19.888081741333007,8.06818962097168,........,null]}}
```

New Cyanite output:

```
[{"id":{"path":"AIO.DIAG.diag2.cpu.cpu.usage","resolution":{"precision":60,"period":2419200},"aggregate":"default"},"time":1496056869,"point":19.888081741333007},{"id":{"path":"AIO.DIAG.diag2.cpu.cpu.usage","resolution":{"precision":60,"period":2419200},"aggregate":"default"},"time":1496056933,"point":8.06818962097168},{"id":{"path":"AIO.DIAG.diag2.cpu.cpu.usage","resolution":{"precision":60,"period":2419200},"aggregate":"default"},....]
```

**Comment by ifesdjeen** [comment]
@avneradania just use 6b3f0a0973df215a69ce65293c3604aa7acb6b9d SHA for now, I'll take a closer look on why that happened.

Sorry for inconvenience.

**Comment by ifesdjeen** [comment]
@avneradania this turned out to be a simple output descripancy: I have pulled one of the functions one level up, but missed out the transformations required. I was using the different setup for testing, so this slipped through. We'll write a couple of integration tests for testing the outputs.

**Comment by ifesdjeen** [comment]
Fixed in #278
